ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -47,11 +49,24 @@ export class PipelineListComponent implements OnInit {
         for (const pipeline of pipelines) {
           pipeline.display = pipeline.name;
         }
+    
         this.pipelineSections = this.processSections(pipelines, 0);
       }
     });
     this.api.queryPipelines();
   }
+  get sortBySelected() {
+    return this.sortByField;
+  }
+
+  set sortBySelected(value) {
+    this.sortByField = value;
+    if (this.sortByField !== 'last_created') {
```

Comment:
Still not sure why this if is needed and why the `last_created` value was added here.